engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -73,6 +79,10 @@ public String apply(@Nullable Level input)
     }
   };
 
+  static {
```

Comment:
The usage of the static block is a general approach in Java development for initialization static properties. I think it is the right way.

MDC properties are static properties. So if both app master and workers are running in the same JVM, MDC properties will have the same values in all threads.